op-chain-ops/upgrades/l1.go

```diff
@@ -617,10 +619,15 @@ func SystemConfig(batch *safe.Batch, implementations superchain.ImplementationLi
 			return err
 		}
 
-		startBlock := common.Hash{}
-
+		var startBlock common.Hash
 		if config != nil {
 			startBlock = common.BigToHash(new(big.Int).SetUint64(config.SystemConfigStartBlock))
+		} else {
+			val, err := strconv.ParseUint(os.Getenv("SYSTEM_CONFIG_START_BLOCK"), 10, 64)
```

Comment:
This implies that it can only be used by one upgrade at a time. I recommend we ensure that it panics if its used for more than 1 upgrade at a time